components/sup/src/census.rs

```diff
@@ -468,27 +466,26 @@ impl CensusGroup {
 pub struct CensusMember {
     pub member_id: MemberId,
     pub pkg: Option<PackageIdent>,
-    application: Option<String>,
-    environment: Option<String>,
-    service: String,
-    group: String,
-    org: Option<String>,
-    initialized: bool,
-    persistent: bool,
-    leader: bool,
-    follower: bool,
-    update_leader: bool,
-    update_follower: bool,
-    election_is_running: bool,
-    election_is_no_quorum: bool,
-    election_is_finished: bool,
-    update_election_is_running: bool,
-    update_election_is_no_quorum: bool,
-    update_election_is_finished: bool,
-    alive: bool,
-    suspect: bool,
-    confirmed: bool,
-    departed: bool,
+    pub application: Option<String>,
+    pub environment: Option<String>,
+    pub service: String,
+    pub group: String,
+    pub org: Option<String>,
+    pub persistent: bool,
+    pub leader: bool,
+    pub follower: bool,
+    pub update_leader: bool,
+    pub update_follower: bool,
+    pub election_is_running: bool,
+    pub election_is_no_quorum: bool,
+    pub election_is_finished: bool,
+    pub update_election_is_running: bool,
+    pub update_election_is_no_quorum: bool,
+    pub update_election_is_finished: bool,
+    pub alive: bool,
```

Comment:
The health portion here (`alive`, `suspect`, `confirmed`, `departed`) should remain private

Reply:
ok. I'll need to remove that info from the `ServiceUpdate` message as well unless we think those fields are going to be needed soon by our subscriber. Currently they are not.

Reply:
ignore that. I can't read.